### DISCUSSION TITLE
CEF 浏览器中无法显示 Tooltip(标签中的 title 属性)

### DISCUSSION THREAD

**Discussion by lxylbds** [CefView/QCefView#411] [Q&A]
在网上找到这个方法，尝试了一下，木有疗效
https://www.cnblogs.com/luckbird/p/cef-tooltip.html
请问还有什么疗法？

**Comment by tishion** [comment]
建议前端自己实现tooltip，或者在使用了QCefView的主项目中添加manifest文件，这不是QCefView项目的职责。
